types/react-native-modal-dropdown/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for react-native-modal-dropdown 0.6
+// Project: https://github.com/nitin42/react-native-modal-dropdown
+// Definitions by: Carlos Li <https://github.com/echoulen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from "react";
+
+import ModalDropdown = RNModalDropdown.ModalDropdown;
+export default ModalDropdown;
```

Comment:
Looks like the package actually [does something a little more than just a default export](https://github.com/sohobloo/react-native-modal-dropdown/blob/master/index.js#L7). To be compatible with both the explicit `default` and the namespace export, you should use `export = ModalDropdown` and then give `ModalDropdown` a static member named `default` of its own type. (eg, `static default: typeof ModalDropdown`)